hedera-node/configuration/mainnet/application.properties

```diff
@@ -3,20 +3,3 @@
 # hedera-config), this requirement will no longer be valid.
 # It's used by modular code for property overrides, taking hedera-config/ as the base, 
 # with overrides from this file (configuration/mainnet/application.properties).
-
```

Comment:
Out of curiosity, what is the reasoning for removing these properties from both bootstrap and application files, but only for mainnet?